docs/faq.md

```diff
@@ -38,22 +38,32 @@
     Submitted batch job 6782
     ```
 
+For more information about how to write and submit jobs to the cluster, visit the [Running Jobs documentation](https://docs.rcc.mcw.edu/user-guide/jobs/running-jobs/){:target="_blank"}.
+
 ??? question "How do I run an interactive job on the cluster?"
-    You can start an interactive job on the HPC cluster using the SLURM `srun` command.
+    You can start an interactive job on the HPC cluster using the SLURM `srun` command. The arguments are the same that you would use in the heather of a script that you would submit as a job. Make sure to add `--pty bash` at the end. To stop the interactive job, use the `exit` command.
```

Comment:
You pointed to the docs in the question above, suggest to do the same here instead of adding info about arguments and job management.

Reply:
If you leave this as is, reword and fix typo.
```suggestion
    You can start an interactive job on the HPC cluster using the SLURM `srun` command. The arguments are the same that you would use in the header of a batch job script. Make sure to add `--pty bash` at the end. To stop the interactive job, use the `exit` command.
```